src/views/Jobs/Jobs.vue

```diff
@@ -13,10 +13,34 @@
         >
           <template v-slot:items="{ item: job }">
             <td>{{ job.product_name }}</td>
-            <td>
+            <td
+              v-if="
+                model(job.model_id) && organism(model(job.model_id).organism_id)
+              "
+            >
               {{ organism(model(job.model_id).organism_id).name }}
             </td>
-            <td>{{ model(job.model_id).name }}</td>
+            <td v-else>
+              <v-progress-circular
```

Comment:
Does it matter if this gets stuck loading forever? e.g. if user runs a job on a new organism, and then deletes that organism

(v-if above nicely fixes an error that blocks the whole page from loading in this example)

Reply:
Yes, there is a dependency issue that is in our todo list. We need to decide how to deal with that, maybe need to delete all dependencies (the same concerns designs when user deletes model that was used for it).